modules/installation-images-samples-disconnected-mirroring-assist.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-restricted-networks-preparations.adoc
+// * openshift_images/samples-operator-alt-registry.adoc
+// * openshift_images/configuring-samples-operator.adoc
+
+[id="installation-images-samples-disconnected-mirroring-assist_{context}"]
+= Cluster Samples Operator assistance for mirroring
+
+During installation, {product-title} creates a config map named `imagestreamtag-to-image` in the `openshift-cluster-samples-operator` namespace. The `imagestreamtag-to-image` config map contains an entry, the populating image, for each image stream tag.
+
+The format of the key for each entry in the data field in the config map is `<image_stream_name>_<image_stream_tag_name>`.
+
+During a disconnected installation of {product-title}, the status of the Cluster Samples Operator is set to `Removed`. If you choose to change it to `Managed`, which overrides the registry to a mirror registry available in your disconnected environment, you can use this config map as a reference for which images need to be mirrored for your image streams to import.
```

Comment:
The last sentence implies setting to 'Managed' overrides the registry.  That is incorrect.  Setting it to 'Managed' only means we will install the samples.  The precise flow is

- while still removed, create your mirrored registry, or determine which existing mirrored registry you want to use
- mirror the samples you want to the mirrored registry using the new configmap as your guide
- add any of the imagestreams you did not mirror to the skippedImagestreams list of the sample operator config object
- set samplesRegistry of the samples operator config object to the mirrored registry
- then finally set to Managed to install the imagestreams you have mirrored 

Now, perhaps we have that sequence of steps elsewhere?  The alt registry flow you've got a ref to below, right?  But let me know if I am incorrect.  I'll defer to you on whether we need to repeat them here, or have this sentence change to pointing to the link where we describe the process.

Reply:
Ah thank you for catching it. I pulled that from the readme with some edits and must have misunderstood. I'm working on some updates. Adding the list and linking back to the _Using Cluster Samples Operator image streams with alternate or mirrored registries_ procedure.